serverV2/contentTypesRouter.js

```diff
@@ -0,0 +1,129 @@
+const express = require('express');
```

Comment:
looks like this file is almost identical to the `contentRouter.js`. can they reuse one each other somehow?